applications/FluidDynamicsApplication/symbolic_generation/compressible_navier_stokes/generate_compressible_navier_stokes_explicit_with_integration.py

```diff
@@ -0,0 +1,303 @@
+from KratosMultiphysics import *
+from KratosMultiphysics.sympy_fe_utilities import *
+
+from sympy import *
+import pprint
+
+from params_dict import params
+import generate_convective_flux
+import generate_diffusive_flux
+import generate_source_term
+import generate_stabilization_matrix
+
+# dim = params["dim"]         # Define Dimension in params.py
+do_simplifications = False
+mode = "c"                  # Output mode to a c++ file
+is_explicit = True          # Explicit or implicit time integration
+
+## Initialize the outstring to be filled with the template .cpp file
+if not is_explicit:
+    template_filename = "compressible_navier_stokes_cpp_template.cpp"
+else:
+    template_filename = "compressible_navier_stokes_explicit_cpp_template_with_integration.cpp"
+templatefile = open(template_filename)
+outstring = templatefile.read()
+
+## Set the output filename
+if template_filename == "compressible_navier_stokes_cpp_template":
+    output_filename = "compressible_navier_stokes.cpp"
+elif template_filename == "compressible_navier_stokes_explicit_cpp_template_with_integration.cpp":
+    output_filename = "compressible_navier_stokes_explicit.cpp"
+else:
+    err_msg = "Wrong template_filename provided. Must be (template --> output):\n"
+    err_msg +=  "\t- compressible_navier_stokes_cpp_template --> compressible_navier_stokes.cpp"
+    err_msg +=  "\t- compressible_navier_stokes_explicit_cpp_template_with_integration.cpp --> compressible_navier_stokes_explicit.cpp"
+    raise Exception(err_msg)
+
+# dim_vector = [2]
+dim_vector = [2,3]
+for dim in dim_vector:
+    # Change dimension accordingly
+    params["dim"] = dim
+
+    # Shape functions and Gauss pts. settings
+    if(dim == 2):
+        n_nodes = 3
+        n_gauss = 3
+    elif(dim == 3):
+        n_nodes = 4
+        n_gauss = 4
+
+    DN = DefineMatrix('DN', n_nodes, dim)
+    mat_N = DefineMatrix('mat_N', n_gauss, n_nodes)
```

Comment:
numeric values of shape functions should be given here

Reply:
Done.